src/components/orders/DetailsTable/index.tsx

```diff
@@ -82,7 +82,19 @@ const tooltip = {
     'The date and time at which the order was submitted. The timezone is based on the browser locale settings.',
   expiration:
     'The date and time at which an order will expire and effectively be cancelled. Depending on the type of order, it may have partial fills upon expiration.',
-  type: 'An order can be either a Buy or Sell order. In addition, an order may be of type "Fill or Kill" (no partial fills) or a regular order (partial fills allowed).',
+  type: (
+    <div>
+      CoW Protocol supports three type of orders – market, limit and liquidity:
+      <ul>
+        <li>A market order is an order to buy or sell at the market&apos;s current best available price</li>
+        <li>A limit order is an order to buy or sell at an arbitrary price specified by the user</li>
+        <li>A liquidity order is an order that market makers can provide as a source of liquidity</li>
+      </ul>
+      In addition, an order can be &quot;fill or kill&quot; or &quot;partially fillable&quot;. Currently all order types
+      can be &quot;fill or kill&quot; – only market orders can be either &quot;fill or kill&quot; or &quot;partially
```

Comment:
This is inaccurate:

```suggestion
      can be &quot;fill or kill&quot; – but only liquidity orders can be either &quot;fill or kill&quot; or &quot;partially
```

Reply:
I flagged this first in slack, and then in the doc I made a proposal https://docs.google.com/document/d/1au3_QDBkzo4CgW9TlmHcQWMEnsQUaH7m0UItgnFEke0/edit#

<img width="752" alt="image" src="https://user-images.githubusercontent.com/2352112/204847880-5313ccfd-0c9d-4c81-ac05-420f4a79f71f.png">


double check with @MindyCoW / @avsavsavs   pls